idl/dsthssp_calib.pro

```diff
@@ -496,7 +495,7 @@ case uvalue of
                  if (i mod 10)  eq 0 then print,i,nhdcam1
                  tmp=min(abs(ref_time[i]-time0),pos)
                  hd[i]=hd0[pos]
-                 ;hd[i].polstate=ref_index[i].polstate
+                 hd[i].polstate=ref_index[i].polstate
```

Comment:
Copying `hd.polstate` header from reference index (camera01's header) to target calibration data's header.